bin/node/src/service.rs

```diff
@@ -204,6 +204,11 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
             .unwrap(),
     );
 
+    assert!(
+        session_period.0 as u64 * 1000 > millisecs_per_block.0,
```

Comment:
session_period is measured in "blocks" not in seconds 
"session_time" would be session_period * MillisecsPerBlock